skactiveml/pool/_bald.py

```diff
@@ -31,44 +31,73 @@ class _GeneralBALD(QueryByCommittee):
 
     Parameters
     ----------
-    n_MC_samples : int > 0, optional (default=n_estimators)
+    n_MC_samples : int > 0, default=n_estimators
         The number of monte carlo samples used for label estimation.
-    greedy_selection : bool, optional (default=False)
+    greedy_selection : bool, default=False
         Flag to either use BatchBALD (`greedy_selection=False`) or a greedy
         (top-k) selection (`greedy_selection=True`) if `batch_size>1`.
-    eps : float  > 0, optional (default=1e-7)
+    eps : float > 0, default=1e-7
         Minimum probability threshold to compute log-probabilities.
-    missing_label : scalar or string or np.nan or None, optional
-        (default=np.nan)
+    sample_predictions_method_name : str, default=None
+        Certain estimators may offer methods enabling to construct a committee
+        by sampling predictions of committee members. This parameter is to
+        indicate the name of such a method.
+            - If `sample_predictions_method_name=None` no sampling is
+              performed.
+            - If `sample_predictions_method_name` is not `None` and in the
+              case of classification, the method is expected to take samples of
+              the shape `(n_samples, *)` as input and to output probabilities
+              of the shape `(n_members, n_samples, n_classes)`, e.g.,
+              `sample_proba` in `skactiveml.base.ClassFrequencyEstimator`.
+            - If `sample_predictions_method_name` is not `None` and in the
+              case of regression, the method is expected to take samples of the
+              shape `(n_samples, *)` as input and to output probabilities of
+              the shape `(n_members, n_samples)`, e.g., `sample_y` in
+              `sklearn.gaussian_process.GaussianProcessRegressor`.
+    sample_predictions_dict : dict, default=None
+        Parameters (excluding the samples) that are passed to the method with
+        the name `sample_predictions_method_name`.
+        - This parameter must be `None`, if
+          `sample_predictions_method_name` is `None`.
+        - Otherwise, it may be used to define the number of sampled members,
+          e.g., by defining `n_samples` as parameter to the method
+          `sample_proba` of `skactiveml.base.ClassFrequencyEstimator` or
+          `sample_y` of
+          `sklearn.gaussian_process.GaussianProcessRegressor`.
```

Comment:
```suggestion
    sample_predictions_dict : dict, default=None
        Parameters (excluding the samples) that are passed to the method with
        the name `sample_predictions_method_name`.
		- This parameter must be `None`, if
		  `sample_predictions_method_name` is `None`.
		- Otherwise, it may be used to define the number of sampled members,
		  e.g., by defining `n_samples` as parameter to the method
		  `sample_proba` of `skactiveml.base.ClassFrequencyEstimator` or
		  `sample_y` of
		  `sklearn.gaussian_process.GaussianProcessRegressor`.
```